### PR TITLE
Add changelog for xdmod11.0

### PR DESCRIPTION
**Pull request by jpwhite4**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+    - Fix bugs in resource specification queries for the internal dashboard. ([\#366](https://github.com/ubccr/xdmod-supremm/pull/366))
+    - Sanitize NAN values from timeseries data ([\#353](https://github.com/ubccr/xdmod-supremm/pull/353))
+
+- Maintenance
+    - Updates to Support PHP7.4 ([\#373](https://github.com/ubccr/xdmod-supremm/pull/373))
+    - Refactor test namespaces and replace redundant exception classes with ones provided by Symfony. ([\#355](https://github.com/ubccr/xdmod-supremm/pull/355))
+    - Remove jQuery as a dependency ([\#380](https://github.com/ubccr/xdmod-supremm/pull/380), [\#377](https://github.com/ubccr/xdmod-supremm/pull/377))
+    - Update jsPlumb dependency. ([\#372](https://github.com/ubccr/xdmod-supremm/pull/372))
+
+- Misc
```
Comment (aaronweeden) [inline CHANGELOG.md:27]:
```suggestion
- Miscellaneous
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,34 @@
...
+- Bug Fixes
+    - Order scatter plot data by usage instead of efficiency statistic.  ([\#323](https://github.com/ubccr/xdmod-supremm/pull/323))
+    - Add LEAST() function to SQL that gets the `wall_time_accuracy` value. ([\#324](https://github.com/ubccr/xdmod-supremm/pull/324))
+    - Use different endpoint for filter values on efficiency tab filter store freeform search.  ([\#325](https://github.com/ubccr/xdmod-supremm/pull/325))
+    - Fix bugs in resource specification queries for the internal dashboard. ([\#366](https://github.com/ubccr/xdmod-supremm/pull/366))
+    - Sanitize NAN values from timeseries data ([\#353](https://github.com/ubccr/xdmod-supremm/pull/353))
+
+- Maintenance
+    - Updates to Support PHP7.4 ([\#373](https://github.com/ubccr/xdmod-supremm/pull/373))
+    - Refactor test namespaces and replace redundant exception classes with ones provided by Symfony. ([\#355](https://github.com/ubccr/xdmod-supremm/pull/355))
```
Comment (aaronweeden) [inline CHANGELOG.md:23]:
```suggestion
    - Replace redundant exception classes with ones provided by Symfony. ([\#355](https://github.com/ubccr/xdmod-supremm/pull/355))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,34 @@
...
+    - Remove jQuery as a dependency ([\#380](https://github.com/ubccr/xdmod-supremm/pull/380), [\#377](https://github.com/ubccr/xdmod-supremm/pull/377))
+    - Update jsPlumb dependency. ([\#372](https://github.com/ubccr/xdmod-supremm/pull/372))
```
Comment (aaronweeden) [inline CHANGELOG.md:25]:
```suggestion
    - Update jsPlumb dependency. ([\#372](https://github.com/ubccr/xdmod-supremm/pull/372))
    - Refactor job data set analysis fields ([\#256](https://github.com/ubccr/xdmod-supremm/pull/256))
```

**Review by aaronweeden — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,34 @@
...
+    - Add GPU Count group by ([\#379](https://github.com/ubccr/xdmod-supremm/pull/379))
+    - Update the efficiency tab for plotly ([\#360](https://github.com/ubccr/xdmod-supremm/pull/360))
+
+- Enhancements
+    - Add new commmunity applications to identification database ([\#361](https://github.com/ubccr/xdmod-supremm/pull/361), [\#376](https://github.com/ubccr/xdmod-supremm/pull/376))
+    - Make `application_id` field available for batch export. ([\#368](https://github.com/ubccr/xdmod-supremm/pull/368))
+
+- Bug Fixes
+    - Order scatter plot data by usage instead of efficiency statistic.  ([\#323](https://github.com/ubccr/xdmod-supremm/pull/323))
+    - Add LEAST() function to SQL that gets the `wall_time_accuracy` value. ([\#324](https://github.com/ubccr/xdmod-supremm/pull/324))
```
Comment (aaronweeden) [inline CHANGELOG.md:16]:
```suggestion
    - Add `LEAST()` function to SQL that gets the `wall_time_accuracy` value. ([\#324](https://github.com/ubccr/xdmod-supremm/pull/324))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,34 @@
...
+
+- Maintenance
+    - Updates to Support PHP7.4 ([\#373](https://github.com/ubccr/xdmod-supremm/pull/373))
+    - Refactor test namespaces and replace redundant exception classes with ones provided by Symfony. ([\#355](https://github.com/ubccr/xdmod-supremm/pull/355))
+    - Remove jQuery as a dependency ([\#380](https://github.com/ubccr/xdmod-supremm/pull/380), [\#377](https://github.com/ubccr/xdmod-supremm/pull/377))
+    - Update jsPlumb dependency. ([\#372](https://github.com/ubccr/xdmod-supremm/pull/372))
+
+- Misc
+    - Use the portal's time zone for raw data query dates instead of UTC. ([\#356](https://github.com/ubccr/xdmod-supremm/pull/356))
+    - Various CI test updates ([\#369](https://github.com/ubccr/xdmod-supremm/pull/369), [\#381](https://github.com/ubccr/xdmod-supremm/pull/381), [\#350](https://github.com/ubccr/xdmod-supremm/pull/350), [\#367](https://github.com/ubccr/xdmod-supremm/pull/367), [\#354](https://github.com/ubccr/xdmod-supremm/pull/354))
```
Comment (aaronweeden) [inline CHANGELOG.md:29]: `xdmod` and the other modules don't list the PRs that change CI tests. Should they, or should this line be removed?

Comment (jpwhite4) [reply CHANGELOG.md:29]: My thought is that the CHANGELOG.md is information for developers and the .html changelog is for end-users. Hence including the ci updates in the .md version

**Pull request merged**